agent/main.go

```diff
@@ -163,6 +165,8 @@ func mainImpl() {
 
 	agentPollURLTemplate := flag.String("agent.poll-url", defaultAgentPollURL, "URL to poll for the agent manifest")
 	agentRecoveryWait := flag.Duration("agent.recovery-wait", defaultAgentRecoveryWait, "Duration to wait before recovering from a failed self update")
+	reportErrors := flag.Bool("agent.report-errors", false, "Should the agent report errors to sentry")
```

Comment:
How come false by default? I would say true by default and we override if we don't want them to be sent, WDYT?

Reply:
It's explicitly enabled in `install.sh` and the `agent.yaml`. We are the only ones likely to run the command manually, so the off-by-default seems like the right choice for us while developing?

Reply:
Makes perfect sense!